DSCResources/MSFT_SystemLocale/MSFT_SystemLocale.psm1

```diff
@@ -0,0 +1,148 @@
+#region LocalizedData
+$Culture = 'en-us'
+if (Test-Path -Path (Join-Path -Path $PSScriptRoot -ChildPath $PSUICulture))
+{
+    $Culture = $PSUICulture
+}
+Import-LocalizedData `
```

Comment:
Importing the localized data is going to cause problems. Since only a 'en-us' strings file is currently provided, the resource may crash on any non-English system. See [this issue](https://github.com/PowerShell/xPSDesiredStateConfiguration/issues/168) from xPSDesiredStateConfiguration.

We worked around this issue in that module by simply commenting out this line and providing the localized data at the top of the file instead.